parser/parser/cc/grammars/typedruby.ypp

```diff
@@ -27,9 +27,7 @@ using namespace std::string_literals;
 %define api.prefix {typedruby27}
 %define api.value.type { union parser_value }
 %define api.token.constructor false
-%define parse.assert
 %define parse.error verbose
-%define parse.trace
```

Comment:
What do you think about leaving a comment here about how we're passing `--debug` in the BUILD file, and not unconditionally in a directive? I can imagine myself grepping for `parse.trace` at some point in the future and being annoyed to not find results.